src/constants/finality.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  arbitrum,
+  avalanche,
+  base,
+  celo,
+  fantom,
+  filecoin,
+  linea,
+  mainnet as ethereum,
+  moonbeam,
+  opBNB,
+  optimism,
+  polygon,
+} from "wagmi/chains";
+
+import { kava } from "./constants";
+
+/** @see https://docs.axelar.dev/learn/txduration#common-finality-time-for-interchain-transactions */
+const finalityTimeMap: Record<string, string> = {
+  [`${ethereum.id}`]: "16 minutes",
+  [`${avalanche.id}`]: "3 seconds",
+  [`${polygon.id}`]: "4:42 minutes",
```

Comment:
I think maybe we should just round this up to 5 minutes

Reply:
@thal0x like this?

```suggestion
  [`${polygon.id}`]: "~5 minutes",
```

Reply:
yep!